src/Tribe/Validate.php

```diff
@@ -227,6 +227,20 @@ public function positive_int() {
 		}
 
 		/**
+		 * validates a field as being an integer
+		 *
+		 * @return stdClass validation result object
+		 */
+		public function int() {
```

Comment:
I don't want to hold this up unnecessarily, so I'm going to drop this in as a musing more than anything.

To me, 'int' is a loaded term. For example, 17179869184 is clearly an integer but it would never be a 'true' integer in terms of the PHP type system if a 32-bit runtime is in use.

Could it be worth renaming the function - or easier yet, expanding on this slightly in the docblock?